### PR TITLE
Added addresable

### PR DESCRIPTION
**Pull request by dipokDipu**
## Summary by CodeRabbit

- **New Features**
	- Enhanced scene loading system with asynchronous operations for loading, setting active, and unloading scenes.
	- Introduced `SceneOperationAction` properties for more flexible scene management.
	- Added new MonoBehaviour scripts for managing scene data and operations.
	- Implemented a variety of build scripts for different modes of asset bundling and resource management.
- **Improvements**
	- Updated GUIDs and metadata across various assets for better organization and management.
	- Refined asset grouping and bundling settings to optimize project builds.
	- Enhanced UI components and interactions for scene loading examples.
- **Bug Fixes**
	- Adjusted scene loading methods to handle multiple scenes and loading screens more effectively.

### CODE REVIEW

File: Assets/Packages/com.studio23.ss2.loadingsystem/Tests/PlayMode/SceneLoadingSystemTests.cs
Comment (zubEpty) [inline Assets/Packages/com.studio23.ss2.loadingsystem/Tests/PlayMode/SceneLoadingSystemTests.cs:1]: enable this commented codes

**Review by zubEpty — APPROVED** [review]

File: Assets/Packages/com.studio23.ss2.loadingsystem/Runtime/Data/SceneData/Operation/AddressableSceneHandle.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+using Cysharp.Threading.Tasks;
+using UnityEngine.AddressableAssets;
+using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.ResourceManagement.ResourceProviders;
+using UnityEngine.SceneManagement;
+
+namespace Studio23.SS2.SceneLoadingSystem.Data
+{
+    public class AddressableSceneHandle
```
Comment (Warhammer4000) [inline Assets/Packages/com.studio23.ss2.loadingsystem/Runtime/Data/SceneData/Operation/AddressableSceneHandle.cs:10]: Should be internal

**Review by Warhammer4000 — CHANGES_REQUESTED** [review]
make the changes

File: Assets/Packages/com.studio23.ss2.loadingsystem/Runtime/Data/SceneData/Operation/AddressableSceneHandle.cs (hunk 1 above)
Comment (dipokDipu) [reply Assets/Packages/com.studio23.ss2.loadingsystem/Runtime/Data/SceneData/Operation/AddressableSceneHandle.cs:10]: Made Internal

File: Assets/Packages/com.studio23.ss2.loadingsystem/Tests/PlayMode/SceneLoadingSystemTests.cs
Comment (dipokDipu) [reply Assets/Packages/com.studio23.ss2.loadingsystem/Tests/PlayMode/SceneLoadingSystemTests.cs:1]: Removed Test Scene for now

**Pull request merged**